File: fibonacci_sequence/fibonacci_sequence.py
from datetime import datetime

def ts():
    return datetime.now().timestamp()

def execution_time(start: datetime.timestamp, end: datetime.timestamp) -> None:
    print(f"{round((end - start) * 1000, 3)} ms")

def fibonacci_sequence(n: int):
    if n < 0: 
        return None
    return fibonacci_sequence(n - 1) + fibonacci_sequence(n - 2) if n > 1 else n

if __name__ == "__main__":
    start = ts()

    fibonacci_sequence(30)

    end = ts()
    execution_time(start, end)

File: fibonacci_sequence/fibonacci_sequence.js
const {performance} = require('perf_hooks');
const ts = () => { return performance.now() }
const execution_time = (start, end) => { console.log(`${(end - start).toFixed(3)} ms`) }

const fibonacci_sequence = (n) => {
    if (n < 0) return Number.NaN
    return n > 1 ? fibonacci_sequence(n -1) + fibonacci_sequence(n -2) : n
}

let st = ts()
fibonacci_sequence(30)
let en = ts()
execution_time(st, en)